--- UVVM-2023.03.21/uvvm_util/src/adaptations_pkg.vhd	2023-03-21 08:23:06.000000000 +0100
+++ UVVM-2023.03.21/uvvm_util/src/adaptations_pkg.vhd	2023-08-08 17:12:19.365786916 +0200
@@ -25,8 +25,8 @@
   constant C_ALERT_FILE_NAME : string := "_Alert.txt";
   constant C_LOG_FILE_NAME   : string := "_Log.txt";
 
-  constant C_SHOW_UVVM_UTILITY_LIBRARY_INFO         : boolean := true; -- Set this to false when you no longer need the initial info
-  constant C_SHOW_UVVM_UTILITY_LIBRARY_RELEASE_INFO : boolean := true; -- Set this to false when you no longer need the release info
+  constant C_SHOW_UVVM_UTILITY_LIBRARY_INFO         : boolean := false; -- Set this to false when you no longer need the initial info
+  constant C_SHOW_UVVM_UTILITY_LIBRARY_RELEASE_INFO : boolean := false; -- Set this to false when you no longer need the release info
 
   constant C_UVVM_TIMEOUT : time := 100 us; -- General timeout for UVVM wait statements
 
@@ -199,7 +199,7 @@
     ID_FUNC_COV_RAND      => DISABLED,
     ID_FUNC_COV_SAMPLE    => DISABLED,
     ID_FUNC_COV_CONFIG    => DISABLED,
-    others                => ENABLED
+    others                => DISABLED
   );
 
   type t_msg_id_indent is array (t_msg_id'left to t_msg_id'right) of string(1 to 4);
@@ -309,7 +309,7 @@
     ID_CHANNEL_BFM           => DISABLED,
     ID_CHANNEL_EXECUTOR      => DISABLED,
     ID_CHANNEL_EXECUTOR_WAIT => DISABLED,
-    others                   => ENABLED
+    others                   => DISABLED
   );
 
   -- Deprecated, will be removed.
@@ -356,7 +356,7 @@
 
   -- Default message Id panel intended for use in SB
   constant C_SB_MSG_ID_PANEL_DEFAULT : t_msg_id_panel := (
-    ID_CTRL => ENABLED,
+    ID_CTRL => DISABLED,
     ID_DATA => DISABLED,
     others  => DISABLED
   );
